src/server/_common.py

```diff
@@ -68,6 +68,7 @@ def log_info_with_request(message, **kwargs):
         remote_addr=request.remote_addr,
         real_remote_addr=get_real_ip_addr(request),
         user_agent=request.user_agent.string,
+        req_referrer=request.referrer or request.origin,
```

Comment:
can we just call this `referrer`?
```suggestion
        referrer=request.referrer or request.origin,
```